### PR TITLE
Governance Clarifications

### PR DESCRIPTION
**Pull request by ekcasey**
Signed-off-by: Emily Casey <ecasey@vmware.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ The purpose of this meeting is to provide in depth support and guidance to the c
 
 ### Team Sync
 
-The purpose of this meeting is to sync across the various teams. They tend to be more technical in nature and go into the "weeds" of on going work.
+The purpose of this meetings is to sync across the various teams. They tend to be more technical in nature and go into the "weeds" of ongoing work.
```
Comment (jkutner) [inline README.md:60]:
```suggestion
The purpose of these meetings is to sync across the various teams. They tend to be more technical in nature and go into the "weeds" of ongoing work.
```

**Review by samj1912 — APPROVED** [review]

File: GOVERNANCE.md [hunk 2]
```diff
@@ -2,53 +2,54 @@
...
+Maintainers are in charge of the day to day maintenance of the team's projects including:
+* ensuring contributions align with project goals and meet the project's quality standards. 
+* reviewing, approving, and merging PRs.
+* planning release milestones, and releasing components under the team's area of responsibility.
+* representing the work of the team to the community.
+* supporting contributors.
+* growing the team by mentoring aspiring contributors and maintainers.
 
-New maintainers must already be contributors, must be nominated by an existing maintainer, and must be elected by a [supermajority](#supermajority) of CNB Team Leads. Likewise, maintainers can be removed by a supermajority of Team Leads or can resign by notifying one of the maintainers.
+New maintainers must already be contributors, must be nominated by an existing maintainer, and must be elected by a [supermajority](#supermajority) of CNB Team Leads and the TOC. Likewise, maintainers can be removed by a supermajority of Team Leads or can resign by notifying one of the maintainers.
```
Comment (samj1912) [inline GOVERNANCE.md:47]:
```suggestion
New maintainers must already be contributors, must be nominated by an existing maintainer, and must be elected by a [supermajority](#supermajority) of CNB Team Leads and the TOC. Likewise, maintainers can be removed by a supermajority of Team Leads and the TOC or can resign by notifying one of the maintainers.
```

[the remark this answers is not in the document]
Comment (ekcasey) [reply GOVERNANCE.md:47]: I'll make it consistent

Comment (ekcasey) [reply GOVERNANCE.md]: resolved in https://github.com/buildpacks/community/pull/166/commits/26201493c1b56dd97faad3eb3bbf977d793df7d2

[the remark this answers is not in the document]
Comment (ekcasey) [reply GOVERNANCE.md]: resolved in https://github.com/buildpacks/community/pull/166/commits/26201493c1b56dd97faad3eb3bbf977d793df7d2

**Pull request merged**